Tests/Resources/app/AppKernel.php

```diff
@@ -12,6 +12,11 @@ class AppKernel extends Kernel
     use MicroKernelTrait;
 
     /**
+     * @var string
+     */
+    private static $cachePrefix;
```

Comment:
nitpick: this is not just the prefix but the whole folder name

Reply:
Thanks.